Backend/Utils/MySqlContext.cs

```diff
@@ -95,15 +95,202 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
-            modelBuilder.Entity<Period>().HasNoKey();
-            modelBuilder.Entity<Occupation>().HasNoKey();
-            modelBuilder.Entity<Shift>().HasNoKey();
+            modelBuilder.Entity<City>().HasData(
+
+                 new City { Id = 21000, Name = "Novi Sad", StateId = 1 },
+                 new City { Id = 11000, Name = "Beograd", StateId = 1 }
+                 );
+
             modelBuilder.Entity<State>().HasData(
-                new State { Id=1, Name="Serbia"}
+                new State { Id = 1, Name = "Serbia" }
+            );
+
+            modelBuilder.Entity<Address>().HasData(
+                new Address { Id = 1, Street = "Radnicka", Number = 4, CityId = 21000 },
+                new Address { Id = 2, Street = "Gospodara Vucica", Number = 5, CityId = 11000 }
                 );
-            modelBuilder.Entity<City>().HasData(
-                new City { Id=21000, Name="Novi Sad", StateId=1}
+
+            modelBuilder.Entity<InsurancePolicy>().HasData(
+                new InsurancePolicy { Company = "Dunav osiguranje d.o.o", Id = "policy1", StartTime = new DateTime(2020, 11, 1), EndTime =  new DateTime(2020, 11, 1) }

```

Comment:
put each variable in single row,so it will easier to read

Reply:
yeah, that would be better